docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,88 @@
+"""
+Coriolis parameters across the world
+====================================
+
+This code computes the Coriolis parameter for an area and stores it in a cube.
+
+It creates and plots a map showing the Coriolis parameters, overlaid on top of
+an Orthographic projection of the globe.
+
+"""
+
+import cartopy.crs as ccrs
+import iris
+from iris.coord_systems import GeogCS
+import iris.plot as iplt
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def main():
+    # Start with arrays for latitudes and longitudes, with a given resolution.
+    resolution = 200
```

Comment:
Good point. I'll need to think a bit about specifically what - the obvious one is "coordinate_size", but I want to avoid any risk of confusion with "cs", which obviously is something else entirely.